tests/Unit/IO/Test_StellarCollapseEos.cpp

```diff
@@ -0,0 +1,376 @@
+  mu_n_data[1][0][1] = -0.010050364713731597;
+  mu_n_data[1][1][0] = -0.011059326583649635;
+  mu_n_data[1][1][1] = -0.010976449692047936;
+  const auto mu_n_from_file = sample_data.get_rank3_dataset("mu_n");
+  CHECK(mu_n_from_file == mu_n_data);
+
+  boost::multi_array<double, 3> mu_p_data(boost::extents[2][2][2]);
+  mu_p_data[0][0][0] = -24.710273945544664;
+  mu_p_data[0][0][1] = -24.710437248365768;
+  mu_p_data[0][1][0] = -24.708423126879229;
+  mu_p_data[0][1][1] = -24.708598930672853;
+  mu_p_data[1][0][0] = -24.710274705168015;
+  mu_p_data[1][0][1] = -24.710440937269624;
+  mu_p_data[1][1][0] = -24.708414929439002;
+  mu_p_data[1][1][1] = -24.708593662516304;
+  const auto mu_p_from_file = sample_data.get_rank3_dataset("mu_p");
+  CHECK(mu_p_from_file == mu_p_data);
+
+  boost::multi_array<double, 3> muhat_data(boost::extents[2][2][2]);
+  muhat_data[0][0][0] = 24.700178644441333;
+  muhat_data[0][0][1] = 24.700418701263001;
+  muhat_data[0][1][0] = 24.697398156068584;
+  muhat_data[0][1][1] = 24.697656836788827;
+  muhat_data[1][0][0] = 24.70014758649021;
+  muhat_data[1][0][1] = 24.700390572555893;
+  muhat_data[1][1][0] = 24.697355602855353;
+  muhat_data[1][1][1] = 24.697617212824255;
+  const auto muhat_from_file = sample_data.get_rank3_dataset("muhat");
+  CHECK(muhat_from_file == muhat_data);
+
+  boost::multi_array<double, 3> munu_data(boost::extents[2][2][2]);
+  munu_data[0][0][0] = -24.193280396074471;
+  munu_data[0][0][1] = -24.193443236769728;
+  munu_data[0][1][0] = -24.190952359387051;
+  munu_data[0][1][1] = -24.191127718625093;
+  munu_data[1][0][0] = -24.192139116306389;
+  munu_data[1][0][1] = -24.192303961055416;
+  munu_data[1][1][0] = -24.189712383229615;
+  munu_data[1][1][1] = -24.18988978156036;
+  const auto munu_from_file = sample_data.get_rank3_dataset("munu");
+  CHECK(munu_from_file == munu_data);
+
+  boost::multi_array<double, 3> r_data(boost::extents[2][2][2]);
+  r_data[0][0][0] = -0.60525665755039704;
+  r_data[0][0][1] = 2.2179733206970909;
+  r_data[0][1][0] = -1.0549307927354015;
+  r_data[0][1][1] = -6.0983303982526049;
+  r_data[1][0][0] = -9.6472871040770141;
+  r_data[1][0][1] = -9.5451610615235349;
+  r_data[1][1][0] = -1.2051288530836537;
+  r_data[1][1][1] = 6.0435070768646284;
+  const auto r_from_file = sample_data.get_rank3_dataset("r");
+  CHECK(r_from_file == r_data);
+
+  boost::multi_array<double, 3> u_data(boost::extents[2][2][2]);
+  u_data[0][0][0] = 9.8002955742002342;
+  u_data[0][0][1] = 2.7463512929327969;
+  u_data[0][1][0] = -2.2719797545844012;
+  u_data[0][1][1] = 0.12238635453091185;
+  u_data[1][0][0] = 1.2619706692408634;
+  u_data[1][0][1] = 4.8905663798386279;
+  u_data[1][1][0] = 7.6749788245333903;
+  u_data[1][1][1] = 7.7886279535935081;
+  const auto u_from_file = sample_data.get_rank3_dataset("u");
+  CHECK(u_from_file == u_data);
+}
+
+SPECTRE_TEST_CASE("Unit.IO.H5.Tabulated", "[Unit][IO][H5]") {
+  test_tabulated("/IO/StellarCollapse2017Sample.h5", "/");
```

Comment:
Use an absolute path to the file. You are making an assumption about where the executable is being run from. Use the `unit_test_path()` function to get the absolute path.